templates/fortinet_fortios_get_router_info_bgp_summary.textfsm

```diff
@@ -1,9 +1,15 @@
 Value BGP_NEIGH (\d+?\.\d+?\.\d+?\.\d+?)
 Value NEIGH_AS (\d+)
-Value UP_DOWN (\w+)
+Value UP_DOWN ((?:\w|:)*)
```

Comment:
Would it be worth it to change to **\S+** to capture anything non-whitespace? This should hopefully prevent us from having to worry too much about any format changes for it.